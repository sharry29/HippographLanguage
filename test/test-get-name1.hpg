int main() {
  node<int> n = 1;
  print_int(n.get_name());
  node<bool> n = true;
  print_bool(n.get_name());
  node<string> n = "foo";
  print(n.get_name());
}