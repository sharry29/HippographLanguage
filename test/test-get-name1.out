1
1
foo